### PR TITLE
New: Show ancestor of item in TreePickerNode component

### PR DESCRIPTION
**Pull request by vinteo**
image

### CODE REVIEW

**Comment by mdotwills** [comment]
LGTM 🎯

File: src/components/adslotUi/TreePickerNodeComponent.jsx [hunk 1]
```diff
@@ -33,6 +33,8 @@ const printAncestorText = (node) => _(node.ancestors)
   .map('label')
   .join(', ');
 
+const pathPrefix = (node) => (_.isEmpty(node.type) ? '' : `${node.type} in `);
```
Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.jsx]: `pathPrefix = ({type}) => ` but I fine with current.

**Comment by vinteo** [comment]
Checked the whitelist and publisher modal, both has not changed

**Comment by OndrejCholeva** [comment]
Merging 🚀

**Pull request merged**